cmd/argo/commands/completion.go

```diff
@@ -13,10 +13,10 @@ func NewCompletionCommand() *cobra.Command {
 		Short: "output shell completion code for the specified shell (bash or zsh)",
 		Long: `Write bash or zsh shell completion code to standard output.
 
-For bash, ensure you have bash completions installed & enabled.
-Either run
+For bash, ensure you have bash completions installed and enabled.
+To access completions in your current shell, run
 $ source <(argo completion bash)
```

Comment:
Sourcing from `.bash_profile` worked for me but `$ source <(argo completion bash)` did not. @mthx can you please suggest what might be the reason? I'm using mac.

Reply:
Silly question, but did you run `source <(./dist/argo-darwin-amd64 completion bash)` to test the local build?

Reply:
I did :) I've also tried `$ ./dist/argo-darwin-amd64 completion bash > /tmp/argo-completion.sh && source /tmp/argo-completion.sh` and it worked perfectly.

Reply:
I think this is a bash version difference.  I'm also on mac.  I usually use zsh but I have OSX bash 3.2.57(1)-release and bash 4.4.19(1)-release from homebrew available to test.  The command I suggested (based on kubectl's docs) works fine on v4 but not v3 (irritatingly they have incompatible bash-completions homebrew packages).

Apparently [it's a bash 3 limitation](https://stackoverflow.com/questions/32596123/why-source-command-doesnt-work-with-process-substitution-in-bash-3-2).  Perhaps just add a note that this is Bash 4 only and flip the order of the two options to suggest writing to a file first?  Bash 3.2 is from 2006(!) but Apple are stuck with it for licensing reasons.

Reply:
If I'm wrong about the version I'll go back to being baffled 😄

Reply:
Thank you for clarifying!